src/utils/styles/_secondary-editor.scss

```diff
@@ -1,37 +1,58 @@
-.yaml-editor__buttons {
```

Comment:
why does the styling name start with _ ? any benefits? , I can see the file is used for the @import, but why the _ ? :) 
we need to try to avoid !important as much as possible. Most of the time, if components don't get inline styling, it's possible due to better selectors... btw I like the mixin and includes

Reply:
Don't know. In my defence I can say: it was already like that :grin: